src/threedidepth/calculate.py

```diff
@@ -84,14 +88,21 @@ def _depth_from_water_level(dem, fillvalue, waterlevel):
 
     @property
     def lookup_s1(self):
+        """
+        Return the lookup table to find waterlevel by cell id.
+
+        Both cells outside any defined grid cell and cells in a grid cell that
+        are currently not active ('no data') will return the NO_DATA_VALUE as
+        defined in threedigrid.
+        """
         try:
             return self.cache[self.LOOKUP_S1]
         except KeyError:
             nodes = self.gr.nodes.subset(SUBSET_2D_OPEN_WATER)
             timeseries = nodes.timeseries(indexes=[self.calculation_step])
             data = timeseries.only("s1", "id").data
             lookup_s1 = np.full((data["id"]).max() + 1, NO_DATA_VALUE)
-            lookup_s1[data["id"]] = data["s1"]
+            lookup_s1[data["id"]] = data["s1"][0]
```

Comment:
`self.lookup_s1` is also used in the `ConstantLevelCalculator`. Does this break because of the change here?

Reply:
No, I noticed that 2D values (1, N) were assigned to a 1D lookup array and made the conversion explicit, that's all. Technically this kind of change should not be part of this pull request, my bad. The time dimension is always one (timestep) long in the context of this library.